Automattic/jetpack
# Blaze: add new method to remotely check for eligibility

## Proposed changes:

This will allow us to remotely check eligibility for a site, and potentially update rules without having to release a new version of the package.

### Other information:


## Jetpack product discussion

* 1386-gh-dotcom-forge

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* In wp-admin, load the Posts list.
* You should see no "Blaze" post action link.
* In your WordPress.com sandbox, remove the check for Jetpack sites in the endpoint (see D98748-code to see how it can be done)
* The "Blaze" post action link should now appear.
